src/platform/plugins/shared/dashboard/public/dashboard_app/listing_page/dashboard_listing_page.test.tsx

```diff
@@ -41,95 +41,71 @@ jest.mock('../no_data/dashboard_app_no_data', () => {
   };
 });
 
-function makeDefaultProps(): DashboardListingPageProps {
-  return {
-    redirectTo: jest.fn(),
-    kbnUrlStateStorage: createKbnUrlStateStorage(),
-  };
-}
-
-function mountWith({ props: incomingProps }: { props?: DashboardListingPageProps }) {
-  const props = incomingProps ?? makeDefaultProps();
-  const wrappingComponent: React.FC<{
-    children: React.ReactNode;
-  }> = ({ children }) => {
-    return <I18nProvider>{children}</I18nProvider>;
-  };
-  const component = mount(<DashboardListingPage {...props} />, {
-    wrappingComponent: wrappingComponent as ComponentType<{}>,
-  });
-  return { component, props };
-}
+const renderDashboardListingPage = (props: Partial<DashboardListingPageProps> = {}) =>
+  render(
+    <DashboardListingPage
+      redirectTo={jest.fn()}
+      kbnUrlStateStorage={createKbnUrlStateStorage()}
+      {...props}
+    />,
+    { wrapper: I18nProvider }
+  );
 
 test('renders analytics no data page when the user has no data view', async () => {
   mockIsDashboardAppInNoDataState.mockResolvedValueOnce(true);
   dataService.dataViews.hasData.hasUserDataView = jest.fn().mockResolvedValue(false);
 
-  let component: ReactWrapper;
-  await act(async () => {
-    ({ component } = mountWith({}));
+  renderDashboardListingPage();
+
+  await waitFor(() => {
+    expect(DashboardAppNoDataPage).toHaveBeenCalled();
   });
-  component!.update();
-  expect(DashboardAppNoDataPage).toHaveBeenCalled();
 });
 
 test('initialFilter is passed through if title is not provided', async () => {
-  const props = makeDefaultProps();
-  props.initialFilter = 'filterPassThrough';
+  const initialFilter = faker.lorem.word();
```

Comment:
Any reason to generate a lorum ipsum word here rather than using some hardcoded string? It's kinda cool, so if that's the reason it works for me 🫡

Reply:
Yeah, it's because it's kinda cool 😂 Initial reason was explained in this [slide](https://docs.google.com/presentation/d/1EgILEx_PQVVF9F4xK1RHr3Q-uBrjLw1Ma2uj8npC5so/edit?slide=id.g2f092009b33_0_233#slide=id.g2f092009b33_0_233), but of course it's not the case here :D 
<img width="881" alt="Screenshot 2025-06-03 at 16 38 11" src="https://github.com/user-attachments/assets/d557525c-f26f-493d-9d59-6c1f8622f2e8" />